packages/components/src/Components/Filters/FilterDropdown.test.js

```diff
@@ -3,15 +3,30 @@ import { shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
 import FilterDropdown from './FilterDropdown';
 
+const TestComponent = () => {
```

Comment:
This is not really a testComponent and allows objects to be rendered, which is not correct. You can redner `<React.Fragment/> ` instead of this `TestComponent`